### ISSUE TITLE
Update this for Meteor 1.3

### ISSUE THREAD

**Issue by talha-asad** [GorillaStack/meteor-c3#2]
``` bash
Unable to resolve some modules:

  "d3" in                                     
/Users/talhaasad/.meteor/packages/gorillastack_c3/.0.4.11.40lpa3++os+web.browser+web.cordova/web.browser/c3/c3.js
(web.browser)
  "jquery" in                                 
/Users/talhaasad/Development/client/lib/plugins/serializeObject/jquery.serialize-object.js
(web.browser)

Consider running: meteor npm install --save d3 jquery
```

**Comment by talha-asad** [comment]
After adding those packages through npm, i still get this in the browser:

``` bash
ReferenceError: c3 is not defined
    at drawChart (template.js:101)
    at Template.c3.rendered (template.js:115)
    at blaze.js?hash=205beea…:3335
    at Function.Template._withTemplateInstanceFunc (blaze.js?hash=205beea…:3681)
    at fireCallbacks (blaze.js?hash=205beea…:3331)
    at .<anonymous> (blaze.js?hash=205beea…:3424)
    at blaze.js?hash=205beea…:1777
    at Object.Blaze._withCurrentView (blaze.js?hash=205beea…:2208)
    at blaze.js?hash=205beea…:1776
    at Object.Tracker._runFlush (tracker.js?hash=b267c37…:515)
```

**Comment by talha-asad** [comment]
One thing particularly missing in this repo is the c3 folder. Also is there a particular reason you do not use the official d3 package on atmosphere?

**Comment by em0ney** [comment]
Hi @talha-asad,

Thanks - I'll check meteor 1.3 compatibility and get back to you shortly.

Regarding your other questions:
- c3 gets included as a git submodule.
- d3 is available through c3.
- This package is designed as a wrapper, to assist with creating graphs with a handlebars helper and being able reactively update the graph as the data changes.

All the best,
Elliott

**Comment by talha-asad** [comment]
Any luck?

**Comment by azrijamilwt** [comment]
d3 package on NPM is not ES6 module, you need a shim to export the d3 module or you can use d3-bundler to export required module. You can use Rollup for better control.

**Comment by em0ney** [comment]
Sorry this has taken so long.  This is resolved in new version 0.4.11-2

Thanks,
Elliott
